File: C/Primer/conditionals.c
// 2019-08
#include <stdio.h>

#define ANSWER 42

int main() {
    int value = 42;

    if (value < 1) {
        // no
    } else if (value == 1) {
        // no
    } else if (value == ANSWER) {
        // yes!
        printf("%d is the answer: %d\n", value, ANSWER);
    } else {
        // no
    }
    // 42 is the answer: 42

    // single line
    printf("%s\n", (value != ANSWER) ? "no" : "this is the answer");
    // this is the answer
}

File: Pascal/Primer/conditionals.pas
// 2020-02
program Conditionals;

const
    ANSWER = 42;
var
    number: integer = 42;
begin
    // single statement in block
    if number = ANSWER then writeln('yes')
    else writeln('no');
    // yes

    // more than one statements in block
    if number = ANSWER then
        begin
            if ANSWER = 42 then
                begin
                    writeln('42 is the answer');
                end
            else
                writeln('42 is not the answer..?');
            // end if
        end
    else
        writeln('no');
    // end if
    // 42 is the answer
end.